assets/media-gallery.js

```diff
@@ -41,11 +41,13 @@ if (!customElements.get('media-gallery')) {
         activeMedia?.classList?.add('is-active');
 
         if (prepend) {
-          activeMedia.parentElement.prepend(activeMedia);
+          activeMedia.parentElement.firstChild !== activeMedia && activeMedia.parentElement.prepend(activeMedia);
+
           if (this.elements.thumbnails) {
             const activeThumbnail = this.elements.thumbnails.querySelector(`[data-target="${mediaId}"]`);
-            activeThumbnail.parentElement.prepend(activeThumbnail);
+            activeThumbnail.parentElement.firstChild !== activeThumbnail && activeThumbnail.parentElement.prepend(activeThumbnail);
           }
+
           if (this.elements.viewer.slider) this.elements.viewer.resetPages();
         }
```

Comment:
I think we should do one other edit. I don't want the theme to scroll me back up to the active media if that active media is the same one as before. 
Below there is a `setTimeout()` in which we're doing the `window.scrollTo()`. We should add a check to not run that in the scenario that we're still showing the same media. 

Here is what I mean: [recording](https://screenshot.click/12-21-p8q9w-lxqsb.mp4)

Though to be fair it was also on previous versions... Maybe something we can double check with UX but I personally find it a bit annoying 🤔

Reply:
Yeah that seems valuable, maybe we roll it out as a separate pr?